app/controllers/admin/participants/school_controller.rb

```diff
@@ -10,6 +10,10 @@ def show
       @school_cohort = @latest_induction_record&.school_cohort
       @lead_provider = @school_cohort&.lead_provider
       @school = @school_cohort&.school
+      @mentees = ParticipantProfile::ECT
+        .merge(InductionRecord.current)
+        .joins(:induction_records)
+        .where(induction_records: { mentor_profile_id: @participant_profile.id })
```

Comment:
Looks good, only a non-blocking question around this doesn't scope the result to the school, but it appears on the school tab - wonder if that's more or less confusing that limiting the scope and potentially not showing all the mentees?